### ISSUE TITLE
Hamburger Navigation Does Not Work

### ISSUE THREAD

**Issue by sgclark** [extendify/extendable#142]
I've installed your theme on my self hosted WordPress install and the 'hamburger' navigation icon does not seem to work.  Within WP Dashboard, I am able to see the navigation in the header block and when I click through to "EDIT" it is pulling the right navigation module.  Not sure if others are seeing the same. I don't have theme active because of this issue. but could do so at a specific time if it helps figure out what is going wrong.

**Comment by sgclark** [comment]
Hi Amjad

It seems that I am using the regular "Header" vs the "Social" header.
Please see screen shot of what I am seeing in WP Dashboard of the different
header options.  I'm using the one with the red border.

[image: Screenshot 2024-04-26 at 12.49.46 PM.png]

Here is a video of the interaction.  I'm clicking on the "Hamburger" menu
and nothing is happening.

https://www.dropbox.com/s/4t4e1amg9rzzvsv/Screen%20Recording%202024-04-26%20at%2012.43.50%E2%80%AFPM.mov?dl=0

The navigation items are:
Home / Wallpapers / Contact Me / About / Archives

The Wallpapers item has numerous 'sub menu' items.

You can go to my site at https://www.sgclark.com to see the navigation now
using my current theme (*not* your Extendable theme)

The Archive page uses a lot of PHP calls via a plug in I use called "PHP
Everywhere" however upon digging into this, it seems this plugin is not
active any more and I may need to replace it or re-build the Archive page.
It is not causing any issues on my current theme (GeneratePress) at this
moment.

Please let me know if you have any further questions.

Stephen

sgc

***@***.***
www.sgclark.com

**Comment by sgclark** [comment]
<img width="2077" alt="Unknown" src="https://github.com/extendify/extendable/assets/13695170/c6db322e-6f79-441e-802d-c92fe7a17bc9">

https://www.dropbox.com/s/4t4e1amg9rzzvsv/Screen%20Recording%202024-04-26%20at%2012.43.50%E2%80%AFPM.mov?dl=0

**Comment by sgclark** [comment]
I'll have to get back to you on the different items you have noted.  I don't see how the navigation functionality has anything to do with what is on the page.  The "Archives" page issue I previously noted is now a non issue as I updated the code and methodology on that page to use 'shortcodes' so I think that should not be an issue any more.

**Comment by amjadr360** [comment]
Yes, the navigation functionality is independent of the content on the page.

Sometimes, one script can conflict with another. To check if other plugins are interfering with the navigation functionality, we could disable the plugins and test the navigation to confirm there are no conflicts.
